scripts/genebuild/create_annotation_configs.pl

```diff
@@ -454,6 +470,20 @@ sub parse_assembly_report {
   $assembly_hash->{'output_path'} .= "/".$species_name."/".$accession."/";
 }
 
+sub update_annotation_status{
+  my $accession = $_[0];
```

Comment:
Could you please add a POD for this method?
We usually use either `my $accession = shift;` or `my ($accession) = @_;`. Using `$_[0]` is correct in Perl but not the way we retrieve argument(s) for a method

Reply:
Changes implemented